Formula/openstackclient.rb

```diff
@@ -6,7 +6,11 @@ class Openstackclient < Formula
   url "https://files.pythonhosted.org/packages/02/0d/18025e48934e59768a085cb4393bebd385be2c4da1d39b4095ab48a321bf/python-openstackclient-5.4.0.tar.gz"
   sha256 "21d228d600421ad5145f31178cdc8578df7d2e569f43381c654caae5a1993b67"
   license "Apache-2.0"
-  revision 3
+  revision 4
+
+  livecheck do
+    url :stable
+  end
```

Comment:
I don't think a `livecheck` block is needed at all for PyPI URLs, actually:
```
❯ brew livecheck --debug openstackclient
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FormulaLoader): loading /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/openstackclient.rb

Formula:          openstackclient
Livecheckable?:   No

URL:              https://files.pythonhosted.org/packages/02/0d/18025e48934e59768a085cb4393bebd385be2c4da1d39b4095ab48a321bf/python-openstackclient-5.4.0.tar.gz
Strategy:         Pypi
URL (strategy):   https://pypi.org/project/python-openstackclient/#files
Regex (strategy): /href=.*?\/packages.*?\/python\-openstackclient[._-]v?(\d+(?:\.\d+)*(?:[._-]post\d+)?)\.t/i

Matched Versions:
5.5.0
openstackclient : 5.4.0 ==> 5.5.0
```
Speaking of which, there's a new version. Would you like to updated it to that so we can drop the revisions?